generators/client/templates/src/main/webapp/app/services/profiles/_active-profiles.service.js

```diff
@@ -0,0 +1,57 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('<%=angularAppName%>')
+        .factory('ActiveProfiles', ActiveProfiles);
+
+    ActiveProfiles.$inject = ['$q', '$resource'];
+
+    function ActiveProfiles($q, $resource) {
+
+        var _activeProfiles;
+
+        var service = {
+            activeProfiles : activeProfiles
```

Comment:
should we call it `getActiveProfiles` ? for aesthetics also we should call the service `ProfileService` rather than `ActiveProfiles`